client-sdk/ts-web/core/test/types.test.ts

```diff
@@ -9,16 +9,20 @@ describe('types', () => {
         it('Should expect fields "amount" and "to"', () => {
             const tw = transferWrapper()
             tw.setBody({ amount, to })
+            expect(tw.transaction.body?.amount).toEqual(amount)
+            expect(tw.transaction.body?.to).toEqual(to)
         })
 
-        it('Should reject other fields', async () => {
+        it('Should detect incorrect fields', async () => {
             const tw = transferWrapper()
             tw.setBody({
                 amount,
                 to,
                 // @ts-expect-error Expect typescript to detect incorrect field
                 otherField: 5,
             })
+            // @ts-expect-error Expect typescript to detect incorrect field
+            expect(tw.transaction.body?.otherField).toEqual(5)
```

Comment:
could this have bone in the previous commit?

Reply:
In the previous commit
```ts
            expect(tw.transaction.body?.amount).toEqual(amount)
            expect(tw.transaction.body?.to).toEqual(to)
            expect(tw.transaction.body?.otherField).toEqual(5)
```
all trigger typescript errors